seed/src/main.rs

```diff
@@ -54,25 +56,25 @@ pub struct Peers {
 /// A set of URNs to track
 #[derive(FromArgs)]
 #[argh(subcommand, name = "track-urns")]
-pub struct Urns {
+pub struct TrackUrns {
     /// track the specified URN only
     #[argh(option, long = "urn")]
     urns: Vec<Urn>,
 }
 
 #[derive(FromArgs)]
 #[argh(subcommand)]
-pub enum Track {
-    Urns(Urns),
-    Peers(Peers),
+pub enum Subcommand {
```

Comment:
```suggestion
pub enum Track {
```

nit: `Subcommand` is not an informative name, `Track` was perfectly fine :)